src/services/students/vitalii-synytskyi.ts

```diff
@@ -0,0 +1,108 @@
+        { x: 5, y: 3, color: Colour.White }, { x: 5, y: 4, color: Colour.White }, { x: 5, y: 8, color: Colour.White }, { x: 5, y: 11, color: Colour.White }, { x: 5, y: 12, color: Colour.White }, { x: 5, y: 13, color: Colour.White },
+        { x: 6, y: 2, color: Colour.White }, { x: 6, y: 4, color: Colour.White }, { x: 6, y: 6, color: Colour.White }, { x: 6, y: 11, color: Colour.White }, { x: 6, y: 14, color: Colour.White },
+        { x: 7, y: 3, color: Colour.White }, { x: 7, y: 4, color: Colour.White }, { x: 7, y: 9, color: Colour.White }, { x: 7, y: 11, color: Colour.White }, { x: 7, y: 12, color: Colour.White }, { x: 7, y: 13, color: Colour.White },
+        { x: 8, y: 4, color: Colour.White }, { x: 8, y: 11, color: Colour.White },
+        { x: 9, y: 1, color: Colour.White }, { x: 9, y: 2, color: Colour.White }, { x: 9, y: 3, color: Colour.White }, { x: 9, y: 4, color: Colour.White }, { x: 9, y: 5, color: Colour.White }, { x: 9, y: 6, color: Colour.White }, { x: 9, y: 7, color: Colour.White }, { x: 9, y: 8, color: Colour.White },
+        { x: 9, y: 9, color: Colour.White }, { x: 9, y: 10, color: Colour.White }, { x: 9, y: 11, color: Colour.White }, { x: 9, y: 12, color: Colour.White }, { x: 9, y: 13, color: Colour.White }, { x: 9, y: 14, color: Colour.White }, { x: 9, y: 15, color: Colour.White },
+        { x: 10, y: 0, color: Colour.White }, { x: 10, y: 15, color: Colour.White },
+        { x: 11, y: 1, color: Colour.White }, { x: 11, y: 2, color: Colour.White }, { x: 11, y: 3, color: Colour.White }, { x: 11, y: 4, color: Colour.White }, { x: 11, y: 5, color: Colour.White }, { x: 11, y: 6, color: Colour.White }, { x: 11, y: 7, color: Colour.White }, { x: 11, y: 8, color: Colour.White },
+        { x: 11, y: 9, color: Colour.White }, { x: 11, y: 10, color: Colour.White }, { x: 11, y: 11, color: Colour.White }, { x: 11, y: 12, color: Colour.White }, { x: 11, y: 13, color: Colour.White }, { x: 11, y: 14, color: Colour.White }, { x: 11, y: 15, color: Colour.White },
+        { x: 12, y: 3, color: Colour.White }, { x: 12, y: 5, color: Colour.White }, { x: 12, y: 10, color: Colour.White }, { x: 12, y: 12, color: Colour.White },
+        { x: 13, y: 3, color: Colour.White }, { x: 13, y: 5, color: Colour.White }, { x: 13, y: 10, color: Colour.White }, { x: 13, y: 12, color: Colour.White },
+        { x: 14, y: 3, color: Colour.White }, { x: 14, y: 5, color: Colour.White }, { x: 14, y: 10, color: Colour.White }, { x: 14, y: 12, color: Colour.White },
+        { x: 15, y: 4, color: Colour.White }, { x: 15, y: 5, color: Colour.White }, { x: 15, y: 6, color: Colour.White }, { x: 15, y: 7, color: Colour.White }, { x: 15, y: 8, color: Colour.White }, { x: 15, y: 9, color: Colour.White }, { x: 15, y: 10, color: Colour.White }, { x: 15, y: 11, color: Colour.White },
+    ];
+
+
+    for (let frameNumber = 0; frameNumber < Constants.MaxTotalFrames; frameNumber++) {
```

Comment:
Really, just 3 frames to represent your entire project ? I demand something more interesting.

Reply:
I can add more frames if needed